tests/test_event_transformer.py

```diff
@@ -375,6 +382,7 @@ def test_multiple_malformed_fields():
     except SnowplowEventTransformationException as sete:
         exception = sete
     assert(exception.error_messages == [
-        "Unexpected exception parsing field with key contexts and value bad_contexts: ValueError('No JSON object could be decoded',)",
+        ("Unexpected exception parsing field with key contexts and value "
+         "bad_contexts: ValueError('No JSON object could be decoded',)"),
```

Comment:
Wow, I didn't know this syntax splitting a string across multiple lines in the source file was legal.